adapter/internal/oasparser/model/api_env.go

```diff
@@ -79,8 +79,10 @@ func retrieveEndpointsFromEnv(apiHashValue string) ([]Endpoint, []Endpoint) {
 //RetrieveEndpointBasicAuthCredentialsFromEnv retrieve endpoint security credentials from env variables
 func RetrieveEndpointBasicAuthCredentialsFromEnv(apiHashValue string, keyType string, endpointSecurity EndpointSecurity) EndpointSecurity {
 	// production Endpoint security
-	endpointSecurity.Username = resolveEnvValueForEndpointConfig(apiHashValue+"_"+keyType+"_basic_username", endpointSecurity.Username)
-	endpointSecurity.Password = resolveEnvValueForEndpointConfig(apiHashValue+"_"+keyType+"_basic_password", endpointSecurity.Password)
+	endpointSecurity.Username = resolveEnvValueForEndpointConfig("api_"+apiHashValue+"_"+keyType+
+		"_basic_username", endpointSecurity.Username)
+	endpointSecurity.Password = resolveEnvValueForEndpointConfig("api_"+apiHashValue+"_"+keyType+
+		"_basic_password", endpointSecurity.Password)
```

Comment:
```suggestion
	endpointSecurity.Username = resolveEnvValueForEndpointConfig("api_" + apiHashValue + "_" + keyType + 
		"_basic_username", endpointSecurity.Username)
	endpointSecurity.Password = resolveEnvValueForEndpointConfig("api_" + apiHashValue + "_" + keyType + 
		"_basic_password", endpointSecurity.Password)
```

Reply:
Fixed

Reply:
im sorry, it seems go doesn't like to have these spaces. vscode automatically reverts the change. shall we undo the commit

Reply:
fixed.